src/test/components/MenuButtons.test.js

```diff
@@ -107,8 +112,10 @@ describe('app/MenuButtons', function() {
       (shortenUrl: any).mockImplementation(() =>
         Promise.resolve('https://profiler.firefox.com/')
       );
-      // Node does not have URL.createObjectURL.
+      // jsdom does not have URL.createObjectURL.
+      // See https://github.com/jsdom/jsdom/issues/1721
       (URL: any).createObjectURL = () => 'mockCreateObjectUrl';
+      (URL: any).revokeObjectURL = () => {};
```

Comment:
Do you think you can do a test to check that revokeObjectURL is actually called? I'd like that to not regress :)
It should be easy if you assign `jest.fn()` here, and then check it's been called with `mockCreateObjectUrl`.

Reply:
Skipping, since I'm going to quickly follow-up.